pkg/kubelet/kubelet.go

```diff
@@ -2483,6 +2489,13 @@ func (kl *Kubelet) canAdmitPod(pods []*api.Pod, pod *api.Pod) (bool, string, str
 			glog.V(2).Infof("Predicate failed on Pod: %v, for reason: %v", format.Pod(pod), message)
 			return fit, reason, message
 		}
+		if re, ok := err.(*predicates.ErrTaintsTolerationsNotMatch); ok {
```

Comment:
You are right, I need to call `PodToleratesNodeTaints` here separately, will fix this